jni/include/jni_util.h

```diff
@@ -17,6 +17,7 @@
 #include <string>
 #include <vector>
 #include <unordered_map>
+#include <cstdint>
```

Comment:
Is this needed?

Reply:
yes, or else it is failing while building the common JNI library with some error like this 
```
'int64_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'
```